docs/bash-completion

```diff
@@ -1,1101 +0,0 @@
-
-# Copyright 2016 The Kubernetes Authors.
```

Comment:
Yeah, I think so.  The bash completion is also autogenerated, and the easiest way for someone to use it will be to generate it on their own with `minikube completion bash > completion`